roles/openshift_metrics/tasks/generate_rolebindings.yaml

```diff
@@ -46,3 +46,75 @@
     files:
     - "{{ mktemp.stdout }}/templates/hawkular-cluster-role.yaml"
     delete_after: true
+
+- name: generate the metrics-server cluster role
+  template:
+    src: metrics-server-role.js2
+    dest: "{{ mktemp.stdout }}/templates/metrics-server-role.yaml"
+  changed_when: no
+
+- name: generate auth-delegator role binding for the metrics-server service account
+  template:
+    src: metrics-server-rolebinding.j2
+    dest: "{{ mktemp.stdout }}/templates/metrics-server-auth-delegator-rolebinding.yaml"
+  vars:
+    cluster: True
+    obj_name: metrics-server:system:auth-delegator
```

Comment:
this is pointlessly parameterized (the whole metrics playbook is).  Just put this structure in the template file -- aim as close to being able to `oc create -f template-dir` as possible.

Reply:
The rest of the playbook relies on and uses the location of those temporary files to do the `oc create`. Do we prefer the pattern used n the service catalog?

    - copy:
        src: kubeservicecatalog_roles_bindings.yml
        dest: "{{ mktemp.stdout }}/kubeservicecatalog_roles_bindings.yml"

    - oc_process:
        create: True
        template_name: service-catalog-role-bindings
        namespace: "kube-service-catalog"

And for remove it uses:

    - oc_obj:
        kind: template
        name: "kube-system-service-catalog-role-bindings"
        namespace: kube-system
        state: absent